source/docs/useful-resources.rst

```diff
@@ -16,25 +16,23 @@ Here is a page dedicated to useful resources around the interwebs. Enjoy!
 General Resources
 -----------------
 
-`FTC Discord <https://discord.com/invite/first-tech-challenge>`_:highlight:`*` --- The **unofficial** FTC\ |reg| Discord server is a discussion-based community server. It is the most active FTC community, which means it generally has the most up-to-date information, and is the easiest way to get quick answers to questions. It also has a channel with direct access to vendors.
+`FTC Discord <https://discord.com/invite/ftc>`_:highlight:`*` --- The **unofficial** FTC\ |reg| Discord server is a discussion-based community server. It is the most active FTC community, which means it generally has the most up-to-date information, and is the easiest way to get quick answers to questions. It also has a channel with direct access to vendors.
 
-.. attention:: The FTC Discord's primary demographic is FTC age participants, which strongly affects the average quality of its discourse, which is often off-topic and occasionally problematic. We recommend mostly sticking to the help channels.
+.. attention:: The FTC Discord's primary demographic is FTC age participants, which strongly affects the average quality of its discourse, which is often off-topic and occasionally problematic. We recommend mostly sticking to the topic-specifc channels and help forums.
 
-`FTC Docs <https://ftc-docs.firstinspires.org>`_:highlight:`*` --- Official *FIRST*\ |reg| FTC documentation, including programming resources, team management resources and more.
+`FIRST Tech Challenge documentation <https://ftc-docs.firstinspires.org>`_:highlight:`*` --- The official *FIRST*\ |reg| Tech Challenge documentation, including programming resources, team management resources, and more.
 
-`FTC Q&A <https://ftc-qa.firstinspires.org>`_:highlight:`*` --- The FTC game Q&A, where clarifying questions about game rules from teams are asked and answered.
+`FIRST Tech Challenge Question and Answer System <https://ftc-qa.firstinspires.org>`_:highlight:`*` --- The official *FIRST* Tech Challenge Question and Answer System, where clarifying questions about game rules from teams are asked and answered.
```

Comment:
maybe tell people to not ask technical questions like how to screw in a screw here?